frontend/src/scenes/pipeline/AppsManagement.tsx

```diff
@@ -0,0 +1,394 @@
+                                    >
+                                        Uninstall
+                                    </LemonButton>
+                                </Popconfirm>
+                            )
+                        },
+                    },
+                ]}
+            />
+        </>
+    )
+}
+
+function OutOfSyncApps(): JSX.Element {
+    const { missingGlobalPlugins, shouldBeGlobalPlugins, shouldNotBeGlobalPlugins } = useValues(appsManagementLogic)
+    if (
+        missingGlobalPlugins.length === 0 &&
+        shouldBeGlobalPlugins.length === 0 &&
+        shouldNotBeGlobalPlugins.length === 0
+    ) {
+        return <></>
+    }
+    return (
+        <>
+            <LemonBanner type="warning"> Apps state is out of sync </LemonBanner>
+            <br />
+            <MissingGlobalPlugins />
+
+            {shouldNotBeGlobalPlugins && (
+                <>
+                    <LemonLabel> Apps that should NOT be global </LemonLabel>
+                    <p>These apps should NOT be global according to repo.</p>
+                    <AppsTable plugins={shouldNotBeGlobalPlugins} />
+                </>
+            )}
+
+            {shouldBeGlobalPlugins && (
+                <>
+                    <LemonLabel> Apps that should be global </LemonLabel>
+                    <p>These already installed apps should be global according to repo.</p>
+                    <AppsTable plugins={shouldBeGlobalPlugins} />
+                </>
+            )}
+            <LemonDivider />
+        </>
+    )
+}
+
+function MissingGlobalPlugins(): JSX.Element {
+    const { missingGlobalPlugins } = useValues(appsManagementLogic)
+    const { installPlugin } = useActions(appsManagementLogic)
+
+    if (missingGlobalPlugins.length === 0) {
+        return <></>
+    }
+    const data = missingGlobalPlugins.map((url: string) => ({ url }))
+    return (
+        <>
+            <LemonLabel> Missing Global Plugins </LemonLabel>
+            <p>These plugins are defined in the GLOBAL_PLUGINS list, but are not installed on this instance.</p>
+            <LemonTable
+                dataSource={data}
+                size="xs"
+                columns={[
+                    {
+                        title: 'URL',
+                        key: 'url',
+                        render: function RenderUrl(_, { url }) {
+                            return (
+                                <Link to={url} target="_blank">
+                                    {url}
+                                </Link>
+                            )
+                        },
+                    },
+                    {
+                        title: 'Install',
+                        key: 'install',
+                        render: function RenderInstallButton(_, { url }) {
+                            return (
+                                <LemonButton
+                                    type="primary"
```

Comment:
There are many of these buttons on one page, so should be secondary. Also small in this context.
```suggestion
                                    type="secondary"
                                    size="small"
```